bin/resources/meshes/Character.cpp

```diff
@@ -1,704 +0,0 @@
-/*/

```

Comment:
This commit is ugly, can you remove it and clean up the other one so that `Character.cpp` doesn't get added to `bin/resources/meshes/Character.cpp`?